ccmlib/scylla_node.py

```diff
@@ -533,10 +533,10 @@ def start(self, join_ring=True, no_wait=False, verbose=False,
                     try:
                         common.check_socket_available(itf)
                     except Exception as msg:
-                        print(f"{msg}. Looking for offending processes...")
+                        print_if_standalone(f"{msg}. Looking for offending processes...", debug_callback=logging.error)
```

Comment:
not in the scope of this change, but we might need to only catch `OSError` and check its `errno` before looking for the offending process.